charts/openyurt/templates/pool-coordinator.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: pool-coordinator-apiserver
-  namespace: {{ .Release.Namespace | quote }}
+  namespace: {{ .Values.poolCoordinator.namespace | quote }}
```

Comment:
I think the target is all components will be installed in the same namespace, so it is convenient to use `.Release.Namespace` to specify namespace for all components, and we don't need a unique namespace for each component.